### PR TITLE
fix(cwl): various fixes

### PR DESCRIPTION
**Pull request by justinmk3**
- Avoid "CWL" in UI.
- Avoid "keyword" in UI. Prefer "Search Log Group", "Search pattern", etc.
- Show (truncated) log group name in search wizard.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/cloudWatchLogs/commands/searchLogGroup.ts [hunk 1]
```diff
@@ -144,10 +145,10 @@ async function logGroupsToArray(logGroups: AsyncIterableIterator<CloudWatchLogs.
     return logGroupsArray
 }
 
-export function createFilterpatternPrompter() {
+export function createFilterpatternPrompter(logGroupName: string) {
     return createInputBox({
-        title: 'Keyword Search',
-        placeholder: 'Enter text here',
+        title: `Search Log Group ${truncate(logGroupName, -50)}`,
```
Comment (justinmk3) [inline src/cloudWatchLogs/commands/searchLogGroup.ts:150]: example:

image

File: src/cloudWatchLogs/commands/searchLogGroup.ts [hunk 2]
```diff
@@ -175,7 +176,7 @@ export class SearchLogGroupWizard extends Wizard<SearchLogGroupWizardResponse> {
         })
 
         this.form.submenuResponse.bindPrompter(createRegionSubmenu)
-        this.form.filterPattern.bindPrompter(createFilterpatternPrompter)
+        this.form.filterPattern.bindPrompter(() => createFilterpatternPrompter(logGroupInfo?.groupName ?? ''))
```
Comment (JadenSimon) [inline src/cloudWatchLogs/commands/searchLogGroup.ts:179]:
```suggestion
        this.form.filterPattern.bindPrompter(({ submenuResponse }) => createFilterpatternPrompter(submenuResponse!.data))
```

You'd want to cover the non-null assertion with a test which I think it already is. I gotta make a PR to explicitly define dependencies to remove these type assertions.

**Review by JadenSimon — APPROVED** [review]

File: src/cloudWatchLogs/commands/searchLogGroup.ts (hunk 1 above)
Comment (Hweinstock) [reply src/cloudWatchLogs/commands/searchLogGroup.ts:150]: I like the idea of showing only the end. That makes it much easier to see what log group we are dealing without overcrowding the user.

**Review by Hweinstock — APPROVED** [review]

**Pull request merged**